gmmp/dashboard_modules.py

```diff
@@ -100,27 +100,27 @@ def init_with_context(self, context):
         self.children = [
             {
                 "name": _("Newspapers"),
-                "count": NewspaperSheet.objects.count(),
+                "count": NewspaperSheet.objects.filter(country=context.request.user.monitor.country).count(),
```

Comment:
Super users should still be able to see everything @ascii-dev ... so probably introduce a method that takes `Model.objects` and add `filter` if user is not a super user similar to the `get_queryset`.

Reply:
Oh... True... I missed that... @kilemensi